modules/web-console/frontend/package.json

```diff
@@ -75,7 +75,7 @@
     "popper.js": "^1.14.7",
     "resize-observer-polyfill": "1.5.0",
     "roboto-font": "0.1.0",
-    "rxjs": "6.4.0",
+    "rxjs": "6.5.2",
```

Comment:
Why you change frontend deps in this ticket?

Reply:
Reverted.